### PR TITLE
Add Russian translation

### PR DESCRIPTION
**Pull request by Ser82-png**

### CODE REVIEW

File: po/ru.po [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+msgid "Refresh interval (minutes)"
+msgstr "Интервал обновления (минуты)"
+
+#: settingsWidget.js:85
+msgid "Hide indicator if there are no devices"
+msgstr "Скрывать индикатор при отсутствии устройств"
+
+#: settingsWidget.js:138
+msgid "Name"
+msgstr "Название "
```
Comment (AlexWayfer) [inline po/ru.po:51]: Why is there a space?

File: po/ru.po [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+msgid "Game Controller"
+msgstr "Игровой контроллер"
+
+#: settingsWidget.js:213
+msgid "Battery"
+msgstr "Аккумулятор"
+
+#: settingsWidget.js:288
+msgid "Python script"
+msgstr "Сценарий Python"
```
Comment (AlexWayfer) [inline po/ru.po:103]: "скрипт" is very common in Russian for familiars with Python, I believe. More common, than "сценарий", what is more… cinematic?

File: po/ru.po (hunk 1 above)
Comment (Ser82-png) [reply po/ru.po]: fix

File: po/ru.po (hunk 2 above)
Comment (Ser82-png) [reply po/ru.po]: fixed to "Скрипт Python"
though using "Сценарий Python" is also valid

**Review by AlexWayfer — APPROVED** [review]

**Pull request merged**